### DISCUSSION TITLE
Tutorial feedback

### DISCUSSION THREAD

**Discussion by nicozhan** [basicai/xtreme1#75] [General]
Tutorial feedback

**Comment by nicozhan** [comment]
The tutorial is very important: it is the very first thing a user sees.
Please report here if anything does not work or is unclear.

Pointing out errors and contributing documentation are always welcome.

**Comment by LuizGuzzo** [comment]
Hello, i'm having some difficulty position the camera corectly on xtreme1. I'm trying to correctly add the camera's extrinsic matrix to calibrate the lidar and camera, but when going to xtreme1 (or basic.ai) it always gives me a really crazy position... I did the test and the cross on the right is the transformation where it should be the camera, however when using it it delivers the camera in a very random position.. and I have no idea how to debug this to make it work correctly, does anyone have any suggestions? Below is the image (I used the print from basic.ai because it shows the camera's line of sight, but the results are the same with the xtreme1)

![image](https://github.com/user-attachments/assets/d834fad6-2c8f-4473-a662-9bc1c906d476)

As you can see the cross on the right and the camera line on the left, it is below the ground and in the opposite direction.. (the other crosses are other sensors)

![image](https://github.com/user-attachments/assets/0206dd62-1455-4e5f-b4d9-14f0dd78c768)

And here is the plot made by another program using the json that I passed to xtreme1/basic.ai showing the correct camera plot.
